tempesta_fw/http.h

```diff
@@ -265,6 +265,7 @@ typedef struct {
 	unsigned long		tm_header;
 	unsigned long		tm_bchunk;
 	unsigned long		hash;
+	__u8			pragma_no_cache:1;
```

Comment:
I just noticed that we process only 3 request types, so 1 byte is enough to keep `method`, so `pragma_no_cache` can be placed near to `method` and we can put both of them into 2 bytes instead of 16 bytes as now (enum is 4 bytes).

Also why not to introduce new TFW_HTTP_CC_PRAGMA_NO_CACHE flag and use `TfwCacheControl`?